test_cases/labels.json

```diff
@@ -70,11 +70,12 @@
     },
     {
       "id": 6,
-      "status": "pass",
+      "status": "fail",
       "issue": "https://github.com/pelias/pelias/issues/294",
```

Comment:
this issue has been closed so a new one needs to opened and referenced here. just change issue to be an array of links.